core/src/components/input/input.tsx

```diff
@@ -263,9 +263,9 @@ export class Input implements ComponentInterface {
 
   /**
    * The size of the input. If "large", it will have an increased height. By default the
-   * size is unset. This property only applies to the `"ionic"` theme.
+   * size is medium. This property only applies to the `"ionic"` theme.
    */
-  @Prop() size?: 'large';
+  @Prop() size?: 'medium' | 'large' = 'medium';
```

Comment:
We might want to create a ticket to track adding default sizes to the other components. Right now we have them set to undefined, this will be the first component that sets a default.

Reply:
Would it be better to change it to undefined?

Reply:
I think we should have a defined size like shape does, we will just have to add the support for it to the other themes and update breaking changes.

Reply:
Okay, parent FW-6216 has been created to track all the changes.

Reply:
Thank you!